README.md

```diff
@@ -10,10 +10,8 @@
 
 This is an interim fork of the excellent project that can be found here: https://github.com/peritus/bumpversion
 
-Unfortunately it seems like development has been stuck for some time and no activity has been seen from the
-author, to that end this project is a drop-in replacement that merges in some of the more important fixes.
-
-Hopefully we can merge back into the original bumpversion and carry on development there.
+In the future, both projects will re-merge with backwards
+compatibility as a goal.
```

Comment:
Should this link to the issue with the intention?

Reply:
Link added.